config/jest/server/stop.mjs

```diff
@@ -1,8 +1,11 @@
-import { teardown } from 'jest-dev-server'
+import { getServers, teardown } from 'jest-dev-server'
 
 /**
  * Stop web server
  */
 export default async function serverStop () {
   await teardown()
+
+  // Clear down server list
+  ;(await getServers()).length = 0
```

Comment:
Bit ugly that StandardJS's "no semicolons" prevents this looking nice

@romaricpascal Let me know if you want it reformatting:

```mjs
// Clear down server list
const servers = await getServers()
servers.length = 0
```

Uses the array `.length = 0` shorthand to delete all items